version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "2.0.0"
+version in ThisBuild := "2.0.1"
```

Comment:
Should probably be `2.1.0` since this is a minor change, not a patch. We should try to follow https://semver.org/ whenever possible.

Reply:
ok 👍